src/js/components/award/shared/AwardPageWrapper.jsx

```diff
@@ -58,6 +59,18 @@ const AwardPageWrapper = ({
                 </span>
             </TooltipWrapper>
             }
+            {unlinked &&
+            <TooltipWrapper
+                tabIndex={0}
+                isVisible
```

Comment:
Looks good but the tooltip opens and closes on hover.  I noticed the AC is incorrect here.  The tooltip should appear on click or tap so it will remain open until the user clicks the close button to exit or clicks the link to jump down to the federal account section.  So you could use a controlled prop here to toggle the tooltip open and closed.

Reply:
changed